doc/classes/Skeleton3D.xml

```diff
@@ -31,6 +31,15 @@
 				[i]Deprecated soon.[/i]
 			</description>
 		</method>
+		<method name="bone_transform_to_world_transform">
+			<return type="Transform">
+			</return>
+			<argument index="0" name="bone_transform" type="Transform">
+			</argument>
+			<description>
+				Takes the given bone pose/transform and converts it to a world transform.
```

Comment:
Which world transform is it?

* The "global transform" from the root node to the bone
* The "world transform" from the skeleton node to the bone. It's probably this one.

Reply:
The latter. I will revise this so it is more clearly stated :+1: